store/interfaces.json

```
@@ -25,7 +29,11 @@
     "component": "ProductBrand"
   },
   "product-images": {
-    "component": "ProductImages"
+    "component": "ProductImages",
+    "composition": "children"
+  },
+  "product-images.main-swiper": {
```

```suggestion
  "product-images-swiper": {
```

I think this block should be called like this. Making it `product-images.*` signals to our platform that it is only a variation of the `product-images` block, which is not the case. An unintended consequence of naming it as you did is that people could use the swiper where only `product-images` blocks were allowed, what is not right and wouldn't work.